lib/transaction_search.rb

```diff
@@ -144,7 +145,8 @@ def handle_csv_search
   end
 
   def email_csv_export
-    AccountTransactionReportMailer.delay.csv_report_email(to_email, @order_details.pluck(:id), @date_range_field)
+    order_detail_ids = @order_details.respond_to?(:pluck) ? @order_details.pluck(:id) : @order_details.map(&:id)
```

Comment:
Is this allowing it to be called with an association or an array? Maybe `@order_details.to_a.map(&:id)`?